config/agencies.localdev.yml

```diff
@@ -1,119 +1,176 @@
 test:
   1:
     name: 'DHS'
+    abbreviation: 'DHS'
```

Comment:
Note that updating this file isn't strictly necessary for this PR but it will make things simpler when we do add the attribute to the schema.